source/speech.py

```diff
@@ -421,6 +422,8 @@ def getIndentationSpeech(indentation):
 		# Translators: This is spoken when the given line has no indentation.
 		return _("no indent")
 
+	#The non-breaking space is semantically a space, so we replace it here.
+	indentation = RE_NBSP_CONVERT.sub(" ", indentation)

```

Comment:
Obviously totally cool with using regexps if it allows you to do multiple string passes in one step and thus improve performance, but in this case, is there any advantage of using a regexp over a simple:

```
indentation = indentation.replace(u"\xa0", " ")
```